packages/flutter/lib/src/material/colors.dart

```diff
@@ -2,7 +2,83 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:ui' show Color;
+import 'dart:ui' show Color, hashValues;
+
+/// A color that has a small table of related colors called a "swatch".
+///
+/// See also:
+///
+///  * [MaterialColor] and [MaterialAccentColor], which define material design
+///    primary and accent color swatches.
+///  * [Colors], which defines all of the standard material design colors.
+class ColorSwatch extends Color {
+  const ColorSwatch(int primary, this.swatch) : super(primary);
```

Comment:
Please doc the constructor too.  We usually restate the class docs briefly.  For example,

```Creates a color that has a small table of related colors called a "swatch".```